index.js

```diff
@@ -3,29 +3,57 @@
  * then logs "Blast Off!"
  */
 
+function countdown() {
+	for (var i = 10; i > 0; i--) {
+		console.log(i);
+	}
+	console.log("Blast Off!");
+}
 
 /**
  * Q1: Write a function called `max` that has two Number parameters and returns
  * the larger of the two.
  */
 
+function max(x,y) {
+	var larger = x > y ? x : y;
+	console.log("max("+ x + "," + y + ") = " + larger);
+	return larger;
+}
 
 /**
  * Q2: Write a function called `longer` that has two String parameters and returns
  * the String that is longer.
  */
 
+function longer(str1,str2) {
+	var longerString = str1.length > str2.length ? str1 : str2;
+	console.log("longer string: " + longerString);
+	return longerString;
+}
+
 
 /**
  * Q3: Write a function called `isEven` that has one Number parameter and returns `true` if
  * if the number is even and `false` otherwise.
- */
+*/
+
+function isEven(n) {
+	var bool = n % 2 === 0 ? true : false;
+	console.log("is " + n + " even? " + bool);
+	return bool;
```

Comment:
You probably know this, but if you didn't care about logging the answer, you could do this all in one line:

```
return n % 2 === 0 ? true : false;
```